Cargo.toml

```diff
@@ -117,6 +117,11 @@ protocol_feature_access_key_nonce_range = ["neard/protocol_feature_access_key_no
 protocol_feature_tx_size_limit = ["near-primitives/protocol_feature_tx_size_limit", "node-runtime/protocol_feature_tx_size_limit", "neard/protocol_feature_tx_size_limit"]
 protocol_feature_allow_create_account_on_delete = ["testlib/protocol_feature_allow_create_account_on_delete", "near-primitives/protocol_feature_allow_create_account_on_delete", "node-runtime/protocol_feature_allow_create_account_on_delete", "neard/protocol_feature_allow_create_account_on_delete"]
 protocol_feature_add_account_versions = ["near-primitives/protocol_feature_add_account_versions"]
+protocol_feature_vm_hash = [
+    "node-runtime/protocol_feature_vm_hash",
```

Comment:
Seems like duplication

Reply:
I had realised that, but just entered them all anyways.